image/templates/helm/stackrox-central/templates/01-central-13-deployment.yaml.htpl

```diff
@@ -112,6 +112,10 @@ spec:
           valueFrom:
             fieldRef:
               fieldPath: metadata.labels['rhacs.redhat.com/tenant']
+        - name: ROX_EXPIRED_AT
+          valueFrom:
+            fieldRef:
+              fieldPath: metadata.annotations['rhacs.redhat.com/expired-at']
```

Comment:
The timestamp value format is not compatible with labels. Regarding restart the opposite is actually true: change in annotations doesn't trigger restart.

Reply:
@0x656b694d it looks like we are both wrong. What I have observed, changing label or annotation does not restart pod. But in this case, we are changing `template`, and changing label or annotation in template will trigger restart.

> The timestamp value format is not compatible with labels.

That is true. But `date +%s` 😉 

If we are not able to avoid restarts, there is no point in changing it. Also, annotation is a better place for such information.